i18n/zh/metadata.yml

```
@@ -0,0 +1,67 @@
+title: 用于奖励开源生态系统的去中心化协议
+abstract: |-
+ 为所有开源软件创建一个开放、公共和稳定的注册表将使项目能够独立发布版本，而不是
+ 依赖于将这些不规则数据组装到数百个独立（和重复）系统中的第三方。Package维护者将
+ 他们的作品发布到由拜占庭容错区块链驱动的去中心化注册表，以消除单一故障源，提供
+ 稳定的发布，并允许社区来管理其开源生态系统的区域，而不受外部议程的影响。
+
+  Tea 通过允许网络参与者将他们依赖并希望保护的软件 Package进行质押来作为激励开源
+  的维护。 Tea 协议的图表提供了不可变的 Package注册、依赖要求、 Package真实性和
+  使用预言机，以告知Tea 报酬算法。系统膨胀基于该算法分配给所有 Package。如果发现
+  安全或开发问题，开发人员可以提出有证据支持的索赔，并可能进行惩罚。开源社区的成员
+  可以审查 Package的质量问题，协议可以通过制定比例削减事件来回应这些审查。
+author:
+- Max Howell
+- Timothy Lewis
+- Thomas Borrel
+references:
+- id: sources
+  url: https://github.com/teaxyz/white-paper
+- id: cc
+  url: https://creativecommons.org/licenses/by-sa/4.0/
+- id: nist
+  url: https://nvd.nist.gov/vuln/detail/CVE-2021-44228
+- id: reuters
+  url: https://www.reuters.com/article/usa-cyber-vulnerability-idCNL1N2SY2PA
+- id: twitter
+  url: https://twitter.com/yazicivo/status/1469349956880408583
+- id: w3
+  url: https://www.w3.org/TR/did-core/
+- id: theregister
+  url: https://www.theregister.com/2016/03/23/npm_left_pad_chaos/
+- id: fossa
+  url: https://fossa.com/blog/npm-packages-colors-faker-corrupted/
+- id: lunasec
+  url: https://www.lunasec.io/docs/blog/node-ipc-protestware/
+- id: github
+  url: https://github.com/dominictarr/event-stream/issues/116
+- id: zdnet
+  url: https://www.zdnet.com/article/open-source-software-how-many-bugs-are-hidden-there-on-purpose/
+- id: threatpost
+  url: https://threatpost.com/backdoor-found-in-utility-for-linux/147581/
+- id: fbi
+  url: https://www.fbi.gov/news/stories/phantom-secure-takedown-031618
+- id: europol
+  url: https://www.europol.europa.eu/media-press/newsroom/news/800-criminals-arrested-in-biggest-ever-law-enforcement-operation-against-encrypted-communication
+- id: medium
+  url: https://medium.com/intrinsic-blog/compromised-npm-package-event-stream-d47d08605502
+- id: semver
+  url: https://semver.org/
+- id: npmjsCrossenv
+  url: https://blog.npmjs.org/post/163723642530/crossenv-malware-on-the-npm-registry.html
+- id: npmjsLodash
+  url: https://www.npmjs.com/package/lodash
+- id: npmjsChalk
+  url: https://www.npmjs.com/package/chalk
+- id: npmjsLogFourjs
+  url: https://www.npmjs.com/package/log4js/
+- id: arxiv
+  url: https://arxiv.org/abs/1207.2617/
+- id: web3
+  url: https://research.web3.foundation/en/latest/polkadot/overview/2-token-economics.html
+lang: zh
+dir: ltr 
+header-includes:
+- \fancyfoot[L]{${VERSION 1}+${zh}}
```

```suggestion
- \usepackage{fancyhdr,ragged2e}
- \lhead{\parbox[t]{0.5\textwidth}{\RaggedRight\rightmark\strut}}
- \rhead{\parbox[t]{0.5\textwidth}{\RaggedLeft\leftmark\strut}}
- \setlength{\headheight}{5\baselineskip}
- \pagestyle{fancy}
- \fancyfoot[LE,RO]{© 2022 tea.inc.}
- \fancyfoot[L]{v1.0.3+zh}
```

add missing header-includes items